pages/common/odps-func.md

```diff
@@ -0,0 +1,19 @@
+# odps func
+
+> Manage functions in ODPS (Open Data Processing Service).
+
+- Show functions in the current project:
+
+`list functions;`
+
+- Create a Java function using a jar resouce:
+
+`create function {{func_name}} as {{path.to.package.Func}} using '{{package.jar}}';`
+
+- Create a Python function using a py resource:
```

Comment:
"py resource" sounds awkward to me. Would ".py file" work? Or we could say ".py resource" (and ".jar resource" above, to match).